src/app/_components/YouTubeVideoEmbed.tsx

```diff
@@ -39,15 +41,23 @@ export function YouTubeVideoEmbed({ videoUrl }: YouTubeVideoEmbedProps) {
   }
 
   return (
-    <LiteYouTubeEmbed
-      id={videoId}
-      title={data.title}
-      wrapperClass="aspect-video bg-cover bg-center rounded-lg overflow-hidden cursor-pointer"
-      iframeClass="focus:brand-outline w-full h-full cursor-pointer"
-      playerClass="sr-only"
-      noCookie={true}
-      params={REMOVE_MORE_VIDEOS_SUGGESTIONS}
-    />
+    <div className="relative">
+      <LiteYouTubeEmbed
+        id={videoId}
+        title={data.title}
+        activatedClass="lyt-activated"
```

Comment:
Maybe group `lyt-activated` and `[.lyt-activated_+_&]:invisible [.lyt-activated_+_&]:opacity-0` in a object?

To highlight their relationship, as we do for touch targets.

Not blocking.

Reply:
Thank you, Charly!